packages/sdk-react/src/components/receivables/InvoiceDetails/ExistingInvoiceDetails/useExistingInvoiceDetails.tsx

```diff
@@ -133,11 +149,26 @@ export function useExistingInvoiceDetails({
   const isDeleteButtonVisible =
     receivable?.status === 'draft' && isDeleteAllowed;
 
+  const isCancelButtonVisible =
+    receivable?.status === 'draft' && isUpdateAllowed;
+
+  const { data: entity } = api.entities.getEntitiesIdSettings.useQuery({
+    path: { entity_id: monite.entityId },
+  });
+
+  const isCancelButtonDisabled =
+    receivable?.status === 'draft' ||
```

Comment:
As I see from the [Invoice lifecycle](https://docs.monite.com/docs/receivables-lifecycle#invoice-lifecycle), you can't Cancel the invoice from the `draft` status. I already asked David for clarification, because the task description is not clear.